src/js/lib/search.js

```diff
@@ -460,11 +460,34 @@ export const getCourseUrl = result => {
     null
 }
 
+export const getSectionUrl = result => {
+  let url = result.url
+  // a small number of urls still start with the legacy site prefix
+  const legacyPrefix = "http://ocw.mit.edu"
```

Comment:
do we have to worry about `http/https` here? if so we might want to 

```js
/https?:\/\/ocw.mit.edu/
```

Reply:
I don't think so actually. I looked through the `ContentFile` objects in the production site and a small number start with `http://ocw.mit.edu`. None start with `https`

Reply:
In that case I think this is fine